Wikipedia/Code/ArticleViewController.swift

```diff
@@ -130,7 +130,9 @@ class ArticleViewController: ViewController, HintPresenting {
     }()
     
     lazy var webView: WKWebView = {
-        return WMFWebView(frame: view.bounds, configuration: webViewConfiguration)
+        let webView = WMFWebView(frame: view.bounds, configuration: webViewConfiguration)
+        view.addSubview(webView)
```

Comment:
This one stops some log spamming - the view wasn't being added to the superview, causing some warnings.